src/app/pages/token/token-detail/token-overview/token-overview.component.ts

```diff
@@ -29,6 +29,10 @@ export class TokenOverviewComponent implements OnInit {
     private environmentService: EnvironmentService,
   ) {}
 
+  ngOnDestroy(): void {
+    this.tokenService.totalTransfer$.next(0);
```

Comment:
Reset lại lúc ban đầu khởi tạo
```suggestion
    this.tokenService.totalTransfer$.next(null);
```